api/setup-jest.ts

```diff
@@ -1,5 +1,8 @@
 import initializeDb from './src/lib/db'
 
 beforeAll(async () => {
+  // avoid conflict in dev container
```

Comment:
I'm not sure this is needed based on the redwood docs here: https://docs.redwoodjs.com/docs/2.x/testing/#the-test-database

can you describe the issue you were noticing a bit more? Maybe we can find an alternate approach that solves it?

Reply:
I removed this change for now

Reply:
@greg-adams can you open a separate PR with this change so we don't lose track of it? I'll have some time tomorrow to review and run the same tests locally to see what might be happening.